Swiftfin/Views/SettingsView/UserDashboardView/DeviceDetailsView/Components/Sections/CustomDeviceNameSection.swift

```diff
@@ -0,0 +1,58 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import JellyfinAPI
+import SwiftUI
+
+// TODO: Enable for CustomNames for Devices: 27, 50, & 53
+
+extension DeviceDetailsView {
+    struct CustomDeviceNameSection: View {
+        var device: DeviceInfo
+
+        private var viewModel = DevicesViewModel()
+
+        @State
+        private var temporaryCustomName: String
+
+        // MARK: - Init
+
+        init(device: DeviceInfo) {
+            self.device = device
+            self.temporaryCustomName = device.name ?? "" // device.customName ?? device.name
+        }
+
+        // MARK: - Body
+
+        var body: some View {
+            Section(L10n.customDeviceName) {
+                ChevronAlertButton(
```

Comment:
Instead, we should have a `TextField` as a row and have an external save button, like user pin hint/password. Have the save button in the top right corner (user password should be migrated to that).

Reply:
I was stealing some of the eventful configuration from the resetUserPasswordViewModel. I'm incorporated that here as well:

https://github.com/user-attachments/assets/4daa5726-0d0f-4a11-a6ca-43c169d21ffd

Reply:
I also added this disclaimer just to try and limit any future issues that might get generated from this. I think after this PR I might make an issue in advance just to get ahead of it:

<img width="326" alt="Screenshot 2024-10-16 at 17 37 43" src="https://github.com/user-attachments/assets/8b1f64a0-8b04-4cce-95dd-ceb2d64a4c04">